examples/keras/train.py

```diff
@@ -0,0 +1,75 @@
+'''Trains and evaluate a simple MLP
+on the Reuters newswire topic classification task.
+'''
+from __future__ import print_function
+
+import numpy as np
+import keras
+from keras.datasets import reuters
+from keras.models import Sequential
+from keras.layers import Dense, Dropout, Activation
+from keras.preprocessing.text import Tokenizer
+import mlflow
+import mlflow.keras
+
+mlflow.keras.autolog()
```

Comment:
Add code comment here to call out that this & the `import mlflow.keras` are the only code changes made to the existing example?